src/Microsoft.DotNet.VersionTools/Automation/DependencyUpdateResults.cs

```diff
@@ -12,22 +14,30 @@ namespace Microsoft.DotNet.VersionTools.Automation
 {
     public class DependencyUpdateResults
     {
+        public IEnumerable<IDependencyInfo> UsedInfos { get; }
+
         public IEnumerable<BuildInfo> UsedBuildInfos { get; }
```

Comment:
I think we can/should get rid of `UsedBuildInfos` and just have `IEnumerable<IDependencyInfo> UsedInfos`.